### PR TITLE
Fixes #25168 - add managed icon interface overview

### PR DESCRIPTION
**Pull request by amitkarsale**

### CODE REVIEW

**Comment by amitkarsale** [comment]
can this be merged?

**Review by xprazak2 — DISMISSED** [review]
Works for me

File: app/assets/javascripts/host_edit_interfaces.js [hunk 1]
```diff
@@ -256,24 +264,28 @@ $(document).on('change', '#host_name', function () {
   update_fqdn();
 });
 
-$(document).on('click', '.primary-flag', function () {
+$(document).on('click', '.provision-flag, .managed-flag, .primary-flag', function () { 
+  var interface_class_substring;
```
Comment (xprazak2) [inline app/assets/javascripts/host_edit_interfaces.js]: nit: training whitespace, previous line as well;

**Review by xprazak2 — DISMISSED** [review]
@amitkarsale, could you rebase please?

**Comment by xprazak2** [comment]
Rebased

**Comment by xprazak2** [comment]
[test foreman]

**Review by xprazak2 — DISMISSED** [review]

**Comment by xprazak2** [comment]
[test foreman]

**Comment by xprazak2** [comment]
[test foreman]

**Review by xprazak2 — APPROVED** [review]

**Comment by xprazak2** [comment]
Thanks!

**Pull request merged**